friends/src/App.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Route, Redirect } from "react-router-dom";
+import { FriendsList } from "./components/FriendsList";
+import { NewFriendForm } from "./components/NewFriendForm";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: []
+    };
```

Comment:
remember that it's not necessary, but since there's nothing being passed into the constructor/super here they're actually superfluous, and you can declare state withing the component itself just as `state = {}`